internal/project/project.go

```diff
@@ -193,7 +193,7 @@ func findSubprojects(superproject Type, apexSuperprojectType projecttype.Type) [
 	var immediateSubprojects []Type
 	for _, subprojectsFolderName := range subprojectsFolderNames {
 		subprojectsPath := superproject.Path.Join(subprojectsFolderName)
-		if subprojectsPath.Exist() {
+		if subprojectsPath.Exist() && subprojectsPath.IsDir() {
```

Comment:
```suggestion
		if subprojectsPath.IsDir() {
```
This check seems redundant, I think we could simply the `subprojectsPath.IsDir()` condition.

Reply:
You are definitely right, `isDir()` checks if the paths exist too